lib/services/local/users.go

```diff
@@ -907,6 +907,16 @@ func (s *IdentityService) GetWebauthnSessionData(ctx context.Context, user, sess
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	if s.Clock().Now().UTC().After(item.Expires) {
+		// Webauthn session already expired. Some backends do not clean up expired
+		// items in a timely manner, force delete.
+		if err := s.Delete(ctx, item.Key); err != nil {
+			s.log.WithError(err).Debugf("Failed to delete expired webauthn session")
+		}
+		return nil, trace.BadParameter("webauthn session expired")
```

Comment:
Should we do an opaque AccessDenied error instead?

Reply:
Hmm I don't think that's necessary, it is the user's webauthn session data so "leaking" that their session expired is fine. From a UX perspective, it makes it clear to the user that their session just expired so they need to retry.